spec/Overview-WebCryptoAPI.xml

```diff
@@ -10083,8 +10083,8 @@ required <a href="#dfn-CryptoKey">CryptoKey</a> <dfn id="dfn-EcdhKeyDeriveParams
                       <ol>
                         <li>
                           <p>
-                            Perform the ECDH primitive specified in <a href="#X9.63">X9.63</a> Section
-                            5.4.1 with <var>key</var> as the EC private key <var>d</var> and the EC public
+                            Perform the ECDH primitive specified in <a href="#RFC6090">RFC6090</a> Section
+                            3 with <var>key</var> as the EC private key <var>d</var> and the EC public
                             key represented by the [[<a href="#dfn-CryptoKey-slot-handle">handle</a>]]
                             internal slot of <var>publicKey</var> as the EC public key <var>Q</var>.
```

Comment:
This needs to be updated, since Q is no longer used.

I'm not even sure Section 4 is an adequate description of ECDH, and whether or not SEC1 should be used rather than 6090. In the SEC1 case, Section 6.1.2 is not appropriate (since Section 6.1.1 requires the selection of a KDF, which is reused in 6.1.3 p3, whereas here, we're completing 6.1.3 p1/2)